mu/app.py

```diff
@@ -114,10 +114,17 @@ def run():
 
     # The app object is the application running on your computer.
     app = QApplication(sys.argv)
+    # By default PyQt uses the script name (run.py)
+    app.setApplicationName('mu')
```

Comment:
Capital "M"... I'll fix this once it's in master.

Reply:
No! It's process name/wmclass, it would be capital M for setDisplayName but we aren't using that because it would append '- Mu' to the window title and "Mu - file.py - Mu" would look a little odd :smile: 

Changes to master mean changing this to `mu-editor` would make sense as long as the .desktop WMClass entry is kept in sync

Reply:
Aha... understood. ;-)